harvester/molecularmatch.py

```diff
@@ -238,7 +238,7 @@ def convert(evidence):
 
     # association['evidence_label'] = direction
     association = el.evidence_label(tier, association, na=False)
-    association = ed.evidence_direction(tier, association, na=False)
+    association = ed.evidence_direction(evidence['direction'], association, na=False)
```

Comment:
this is not returning the correct value to `response_type`. There are multiple different values from the source with different contexts which should be preserved (for now):

```python
{
    'conflicting': 35,
    'decreased response': 394,
    'no benefit': 265,
    'not applicable': 69,
    'predicted – resistant': 45,
    'predicted – sensitive': 507,
    'resistant': 976,
    'sensitive': 3449,
    'unknown': 6
}
```

Reply:
@ahwagner :  where did you find these responses in molecularmatch data?

Reply:
```
cat /tmp/molecularmatch_raw.json  | jq .direction  | sort | uniq
"does_not_support"
"supports"
```

Reply:
I didn't save the code, and now I'm not sure. Looking at MM records, what we want for this field is clearly a combination of top level keys ['direction'] and ['clinicalSignificance']. I'm wondering if I was looking at a different source? Anyway, I'm trying to verify that molecularmatch was the source with the entries that needed adjusting, but it's not loading for me right now (timeout errors). I'll try again tonight and hopefully can get you some more info.

Reply:
@ahwagner  Re. molecularmatch

```
association['response_type'] = '{} {}'.format(evidence['direction'], evidence['clinicalSignificance'])
``` 
### before
![image](https://user-images.githubusercontent.com/47808/42353079-1f5adaaa-8073-11e8-9b31-dc76eb8f0afe.png)

### after
![image](https://user-images.githubusercontent.com/47808/42353116-59645410-8073-11e8-8900-dafac0ccd7a9.png)